contents/docs/error-tracking/installation/web.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Web error tracking installation
+---
+
+import { Steps, Step } from 'components/Docs/Steps'
+
+import Snippet from "../../integrate/_snippets/install-js-snippet.mdx"
+import InstallWebPackageManagers from "../../integrate/_snippets/install-web-package-managers.mdx"
+
+<Steps>
+<Step title="Installing PostHog web SDK" subtitle="Your goal in this step: Install the PostHog web JavaScript SDK." badge="required">
+
+### Option 1: Add the JavaScript snippet to your HTML <span class="bg-gray-accent-light dark:bg-gray-accent-dark text-gray font-semibold align-middle text-sm p-1 rounded">Recommended</span>
+
+<Snippet />
+
+### Option 2: Install via package manager
+
+<InstallWebPackageManagers />
+
+And then include it with your project API key and host (which you can find in [your project settings](https://app.posthog.com/settings/project)):
+```js-web
+import posthog from 'posthog-js'
+
+posthog.init('<ph_project_api_key>', {
+  api_host: '<ph_client_api_host>',
+  defaults: '<ph_posthog_js_defaults>'
+})
+```
+
+See our framework specific docs for [Next.js](/docs/libraries/next-js), [React](/docs/libraries/react), [Vue](/docs/libraries/vue-js), [Angular](/docs/libraries/angular), [Astro](/docs/libraries/astro), [Remix](/docs/libraries/remix), and [Svelte](/docs/libraries/svelte) for more installation details.
+
+<details>
+<summary>Bundle all required extensions (advanced)</summary>
```

Comment:
I'm not sure this is necessary for here and I'm worried we are duplicating content between here and the main JavaScript install guide so this will go out of date.